docs/faqs/NuGet-FAQ.md

```diff
@@ -232,15 +232,17 @@ If that fails, try the [tips mentioned in this StackOverflow post](http://stacko
 
 ## What is the API endpoint for nuget.org?
 
-To use  nuget.org as a package repository with NuGet clients, you would need to use the following V3 API endpoint: 
+To use nuget.org as a package repository with NuGet clients, you would need to use the following V3 API endpoint: 
 
 **`https://api.nuget.org/v3/index.json`**
 
-Older clients can still use the v2 protocol to reach nuget.org. However, please note, NuGet clients 3.0 or later will have slower and less-reliable service using the v2 protocol:
+Older clients can still use the V2 protocol to reach nuget.org. However, please note, NuGet clients 3.0 or later will have slower and less-reliable service using the V2 protocol:
 
-`https://www.nuget.org/api/v2/` (DEPRECATED!!!) **Note:** the "www." is important.
+`https://www.nuget.org/api/v2/` (DEPRECATED!!!) **Note:** use "www." for the best reliability.
 
-Additionally, *NuGet.exe List* only works with the v2 protocol.
+Additionally, *nuget.exe list*, *push* and *delete* work using V2 endpoints whether the client is configured
```

Comment:
While this is  fine, the intent here was to clarify that the `nuget list` command is broken if a server implements just documented APIs (V3 + V2 publish APIs). I think that this new paragraph no longer conveys that. Thoughts?

Reply:
This isn't strictly true. You could say the endpoints used by nuget.exe list are both V2 and V3 endpoints. The fact that they are OData via a resource called "LegacyGallery" is an implementation detail. What real life scenario are we trying to protect users from?

Reply:
Additionally, we could decouple the endpoints simply by modifying our V3 service index to point to new locations.

Reply:
if you wanted to have some docs for server implementers, that kind of detail would be fine there? this FAQ is meant to cover how nuget client users should configure the client.
the fact that push, delete and list all get redirected to v2 URIs under the covers is likely irrelevant. I could live without any mention of this fact in this section of the doc.
THoughts?

Reply:
I agree. I would remove the V2 references from this doc. They may be applicable in the API docs not here so much.

Reply:
Sure, i'll remove this sentence entirely and enhance the API docs to talk about the `nuget.exe list` case.

Reply:
k. sounds good.